vultr-hook.sh

```diff
@@ -54,17 +58,16 @@ clean_challenge() {
     ENTRY="_acme-challenge.${HOST}"
 
 read -r -d '' FILTER <<- EOF
-.[] | if .name == "${ENTRY}" then .RECORDID else empty end
+.records[] | if .name == "_acme-challenge" then .id else empty end
```

Comment:
This will only match `_acme-challenge` record instead of the `_acme-challenge.subdomain` challenge that has been created.

So this will not remove the just created verification entry.

Reply:
Good catch, updating this too.